opm/input/eclipse/Schedule/Schedule.hpp

```diff
@@ -515,7 +516,8 @@ namespace Opm
                            const ParseContext& parseContext_,
                            ErrorGuard& errors_,
                            SimulatorUpdate * sim_update_,
-                           const std::unordered_map<std::string, double> * target_wellpi_)
+                           const std::unordered_map<std::string, double> * target_wellpi_,
+                           std::unordered_map<std::string, double>* wellpi_global_factor_ = nullptr)
```

Comment:
I think the default value for argument `wellpi_global_factor_` can be removed here.  But I will address it with other potential comments.